stretch_core/launch/upright_camera_view.launch

```diff
@@ -0,0 +1,18 @@
+<launch>
```

Comment:
This launch file isn't used anywhere. Should it be included in Stretch Core and Stretch Gazebo?